python/paddle/fluid/layers/nn.py

```diff
@@ -770,6 +780,11 @@ def dynamic_lstmp(input,
                               the bias is initialized zero. Default: None.
         use_peepholes(bool): Whether to enable diagonal/peephole connections,
                              default `True`.
+        cell_clip(float): If provided the cell state is clipped
```

Comment:
How about another name like `cell_clip_threshold`, and adding an assert `assert(cell_clip_threshold>=0)` ?

Reply:
this parameter uses the same name in API of tensorflow rnn cell, so the name may be more convenient to user.

Reply:
can add assert(cell_clip >=0)